### PR TITLE
Refactor linux install

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: ziti/cmd/create/create_config_environment.go [hunk 1]
```diff
@@ -187,47 +188,47 @@ func NewCmdCreateConfigEnvironment() *cobra.Command {
 		"environment variables are used. All settings have default values but can be manually set to override " +
 		"the config output.\n\nThe following environment variables can be set to override config values " +
 		"(current value is displayed):\n")
-	sb.WriteString(fmt.Sprintf("%-40s %-50s %s\n", constants.ZitiHomeVarName, constants.ZitiHomeVarDescription, data.ZitiHome))
```
Comment (qrkourier) [inline ziti/cmd/create/create_config_environment.go:190]: Please double check tidying here.  This is the environment usage hint output and the removed strings didn't seem to be doing anything (not printed).

File: dist/dist-packages/linux/openziti-controller/bootstrap.bash [hunk 2]
```diff
@@ -166,49 +166,265 @@ function makeDatabase() {
...
+: "${DEBUG:=0}"
+if (( DEBUG )); then
+  exec 3>&1
+  set -o xtrace
+else
+  exec 3>/dev/null
+fi
+
+
+# set global defaults applicable to bootstrapping and normal operation
```
Comment (qrkourier) [inline dist/dist-packages/linux/openziti-controller/bootstrap.bash:385]: defaults relocated from answer file

File: dist/dist-packages/linux/openziti-controller/bootstrap.bash [hunk 3]
```diff
@@ -166,49 +166,265 @@ function makeDatabase() {
...
+: "${ZITI_HOME:=/var/lib/ziti-controller}"; export ZITI_HOME
+: "${ZITI_PKI_ROOT:=pki}"  # relative to systemd service WorkingDirectory; e.g., /var/lib/ziti-controller/pki
+: "${ZITI_CA_FILE:=root}"  # relative to ZITI_PKI_ROOT; root CA dir; e.g., /var/lib/ziti-controller/pki/root
+: "${ZITI_INTERMEDIATE_FILE:=intermediate}"  # intermediate CA dir; e.g., /var/lib/ziti-controller/pki/intermediate
+: "${ZITI_SERVER_FILE:=server}"  # relative to intermediate CA "keys" and "certs" dirs
+: "${ZITI_CLIENT_FILE:=client}"  # relative to intermediate CA "keys" and "certs" dirs
+: "${ZITI_NETWORK_NAME:=ctrl}"  # basename of identity files
+
+# run the bootstrap function if this script is executed directly
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
```
Comment (qrkourier) [inline dist/dist-packages/linux/openziti-controller/bootstrap.bash:395]: enclosed commands are only executed on Linux, never in Docker.

**Comment by qrkourier** [comment]
Marked not ready for review because I found a side effect that needs fixing.

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**